README.md

```diff
@@ -39,14 +48,38 @@ You need an access token to publish, install, and delete private, internal, and
 
 You can use a personal access token (classic) to authenticate to GitHub Packages or the GitHub API. When you create a personal access token (classic), you can assign the token different scopes depending on your needs. 
 
+Login and access token can be set in Gradle's configuration [build.gradle](build.gradle) file:
+```
+credentials {
+    username "your_username"
+    password "your_github_access_token"
+}
+```
+To access [Java Intraday API package](https://maven.pkg.github.com/NordPool/public-intraday-api) you will need privileges that can be obtained from [idapi@nordpoolgroup.com](mailto:idapi@nordpoolgroup.com).
+
 ## Important considerations ##
 
 The current program is using the Spring library and its Websocket Stomp client. The usage of it is at your own discretion. 
 
-In **com/nordpool/intraday/publicapi/example/service/connection/WebSocketConnector.java** please note  the configuration of maxTextMessageSize. Some messages may be quite large. 
+In addition, sending heartbeat task created after connection established and refreshing access token are  defined in the WebSocketConnector. That connector can be found from [WebSocketConnector.java](src/main/java/nps/id/publicapi/java/client/connection/WebSocketConnector.java).
+Heartbeat interval configuration can be found in [application.properties](src/main/resources/application.properties) **market.data.heartbeat.outgoing.interval** property for Market data web service and **trading.heartbeat.outgoing.interval** for Trading web service.
+
+In [WebSocketConnector.java](/src/main/java/nps/id/publicapi/java/client/connection/WebSocketConnector.java) please note the configuration of **maxTextMessageSize** and **maxByteMessageSize**. 
+Some messages may be quite large. 
+
+The example uses ports 8083/443(secured) for establishing the web socket connection with **Trading** web service and ports 80/443(secured) for establishing web socket connection with **Market data** web service. 
```

Comment:
port 8083 is not available, we are exposing 443 only. Please remove any sign of insecure connection from example
trading.web.socket.useSsl must always be true

Reply:
Sure, I assume should do the same with .net example? 
There is also possibility to configure useSsl parameter.

Reply:
not really, all our connections are secure, this flag does not make much sense, so it should be removed form both examples